src/content/docs/en/guides/images.mdx

```diff
@@ -679,7 +679,7 @@ It returns an object with the following properties:
 
 Not all users can see images in the same way, so accessibility is an especially important concern when using images. Use the `alt` attribute to provide [descriptive alt text](https://www.w3.org/WAI/tutorials/images/) for images.
 
-This attribute is required for the `<Image />` component. `<Image />` will throw an error if no alt text is provided.
+This attribute is required for the `<Image />` and `<Picture />` components. `<Image />` and `<Picture />` will throw an error if no alt text is provided.
```

Comment:
```suggestion
This attribute is required for both the `<Image />` and `<Picture />` components. If no alt text is provided, a helpful error message will be provided reminding you to include the `alt` attribute.
```
How about something like this, to avoid some text duplication?

Reply:
Makes sense to me!